cumulus/provider.tf

```diff
@@ -13,8 +13,7 @@ terraform {
       version = "~> 2.2.0"
     }
   }
-  backend "s3" {
-  }
+  backend "s3" {}
 }
 
 provider "aws" {
```

Comment:
Should probably still put it here even though there are no resources in this module at the moment

Reply:
Decided against it as this module doesn't even have a variables or locals file.